src/winter_onboarding_2021/4clojure/alisha.clj

```diff
@@ -2,4 +2,4 @@
 
 (defn problem4 [] (list :a :b :c))
 
-
+(defn problem5 [] (conj '(2 3 4) 1))
```

Comment:
Use better commit message here. 
- You've not added a function called `conj`
- The commit message doesn't tell me much about the change.